src/query/api/v1/handler/prometheus/remote/write.go

```diff
@@ -557,6 +559,10 @@ func (h *PromWriteHandler) parseRequest(
 		}
 	}
 
+	if clusterType := r.Header.Get(headers.ClusterTypeHeader); clusterType != "" {
+		h.metrics.writeScope.Tagged(map[string]string{"kaas_type": clusterType}).Counter("total").Inc(1)
```

Comment:
there is already a cluserType label, use this to avoid conflict

Reply:
some of them may be not KaaS, how about call clientType?

Reply:
sounds good